src/Compilers/Core/Portable/DiagnosticAnalyzer/DefaultAnalyzerAssemblyLoader.cs

```diff
@@ -12,6 +12,10 @@
 using System.Runtime.InteropServices;
 using Roslyn.Utilities;
 
+#if NET
```

Comment:
need to go over this with @jaredpar .  I'm also happy to extract into its won PR to make thigns simpelr.